administrator/components/com_finder/src/Indexer/Indexer.php

```diff
@@ -946,38 +953,15 @@ protected function addTokensToDb($tokens, $context = '')
 					. (int) $context . ', '
 					. $db->quote($token->language)
 				);
-				$values++;
-
-				if ($values > 0 && ($values % 128) == 0)
-				{
-					$db->setQuery($query)->execute();
-					$query->clear('values');
-
-					// Check if we're approaching the memory limit of the token table.
-					if ($values > static::$state->options->get('memory_table_limit', 10000))
-					{
-						$this->toggleTables(false);
-					}
-				}
+				++$values;
 			}
```

Comment:
```suggestion
			}

```
Unfortunately PHPCS is still not happy because the previous suggestion removed the complete line and not the spaces only. Maybe my mistake.